modules/server/src/main/scala/scaladex/server/route/FrontPage.scala

```diff
@@ -69,18 +61,146 @@ class FrontPage(env: Env, database: WebDatabase, searchEngine: SearchEngine)(imp
       val sbtVersions = platforms.collect { case (v: SbtPlugin, c) => (v, c) }.sorted(RecentPlatformVersionsFirst)
       val millVersions = platforms.collect { case (v: MillPlugin, c) => (v, c) }.sorted(RecentPlatformVersionsFirst)
 
+      def selectCurrent(allVersions: Seq[EcosystemVersion]): Option[(EcosystemVersion, Seq[EcosystemVersion])] =
+        allVersions.filterNot(_.deprecated).headOption.map { currentVersion =>
+          val otherVersions = allVersions.filterNot(_ == currentVersion)
+          currentVersion -> otherVersions
+        }
+
+      val scalajsEcosystem = {
+        val allVersions = scalaJsVersions.map {
+          case (sjs, cnt) =>
+            EcosystemVersion(
+              version = sjs.version,
+              deprecated = sjs.isDeprecated,
+              libraryCount = cnt,
+              search = Url(s"search?platforms=${sjs.label}")
+            )
+        }
+        selectCurrent(allVersions).map {
+          case (currentVersion, otherVersions) =>
+            EcosystemHighlight(
+              "Scala.js",
+              Some("Robust Scala-to-JavaScript compiler for browser and node.js."),
```

Comment:
I'm not sure if these taglines for Scala.js and Scala Native are helping in this context. Removing them would declutter a bit more, IMO.